amqp_influxdb/__init__.py

```diff
@@ -21,12 +21,15 @@
 import requests
 import pika
 from pika.exceptions import AMQPConnectionError
+import ssl
```

Comment:
stand lib imports should be separated from 3rd party imports and come before them. 
```
import ssl

import requests
import pika
from pika.exceptions import AMQPConnectionError
```